content/topics/file-and-directory-naming/python/_index.md

```diff
@@ -0,0 +1,50 @@
+---
+content_type: topic
+prerequisites:
+  hard:
+  - topics/file-and-directory-naming/general
+ready: true
+tags:
+- File and directory naming
+- python
+title: File and directory naming in Python
+---
+
+## How should I name my Python modules and packages?
+
+You should name your files and directories the same way you name Python variables.They should be small letters, and words should be separated by underscores, as that improves readability.
+
+It is important to note that the use of underscores in directory_naming takes preference as opposed to the use of dashes.
+
+Also, be sure to keep in mind that all Python files have a .py extension. See below the recommended Python file and directory naming style. 
+
+### NO:
+```
+TASK
+    1task.py
+    task 1.py
+    task1.1.py
+        
+passwordChecker
+    add-logger.py
+
+snakecase
+    productionnotes.py
+    keepItShortAndmeaningful.py
+```
+
+### Yes:
+
+```
+task
+    task_1.py
+    task_2.py
+    task_1_1.py
+
+password-checker
```

Comment:
since we say that underscores take preference, should this one not be an underscore?

Reply:
There is an example of underscore use below

Reply:
Do you want this restructured?

Reply:
I guess it shows that both are acceptable, even if one is preferable

Reply:
We prefer underscores. You can't say `import password-checker` in python.